docs/standard/glossary.md

```diff
@@ -0,0 +1,228 @@
+---
+title: .NET Glossary
+description: Find out the meaning of selected terms used in the .NET documentation.
+keywords: .NET glossary, .NET dictionary, .NET terminology, .NET platform, .NET framework, .NET runtime
+author: tdykstra
+ms.author: tdykstra
+ms.date: 07/08/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+ms.devlang: dotnet
+---
+
+# .NET Glossary
+
+The primary goal of this glossary is to clarify meanings of selected terms and acronyms that appear frequently in the .NET documentation without definitions. Some of these terms are used inconsistently, so a secondary goal is to help reduce the confusion that inconsistent usage causes.
+
+## AOT
+
+Ahead-of-time compiler. Similar to JIT, this compiler also translates IL to machine code. In contrast to JIT compilation, AOT compilation happens before the application is executed and is usually performed on a different machine. AOT tool chains don't trade runtime for compile time and thus can spend more time optimizing. Since the context of AOT is the entire application, the AOT compiler can also perform cross-module linking and whole-program analysis, which means that all references are followed and a single executable is produced.
```

Comment:
In starting with a noun that describes the entry I'm following the lead of the [Azure Glossary](https://docs.microsoft.com/en-us/azure/azure-glossary-cloud-terminology).  I'll leave it now but have asked Monica for the rationale behind the Azure Glossary style.

Putting def on its own line is worth a try.

I don't think we've adopted as a style standard the practice of reducing every occurrence of "application" to "app" but I'll look for additional places where the shortened form might fit better.